source/client/process_impl.cc

```diff
@@ -179,7 +181,10 @@ void ProcessImpl::createWorkers(const uint32_t concurrency) {
   // TODO(oschaaf): Arguably, this ought to be the job of a rate limiter with awareness of the
```

Comment:
I think this TODO has become out of date with our ideal, given this restructure.  Do we still want a global rate limiter to do this work anymore, given what we now know?  If not, can we remove this TODO?